package.json

```diff
@@ -40,6 +42,10 @@
 			],
 			"multiline-comment-style": "off",
 			"computed-property-spacing": "off",
+			"space-before-function-paren": [
```

Comment:
hmm this must have gotten removed somewhere?

Reply:
Seems like it, this settings seems to be what we're aiming though.